### PR TITLE
Radar center label text by default

### PR DESCRIPTION
**Pull request by zachguo**
Before:
screen shot 2017-06-30 at 2 00 29 pm
After:
screen shot 2017-06-30 at 2 10 35 pm

### CODE REVIEW

File: showcase/radar-chart/animated-radar-chart.js [hunk 1]
```diff
@@ -72,6 +72,9 @@ export default class AnimatedRadar extends Component {
               text: {
                 opacity: 0
               }
+            },
+            labels: {
```
Comment (mcnuttandrew) [inline showcase/radar-chart/animated-radar-chart.js]: not necessary if this is the new default?

Comment (zachguo) [inline showcase/radar-chart/animated-radar-chart.js]: It's just for showing the effect, or the default `labels` style will be override by an empty `{}`.

Comment (mcnuttandrew) [inline showcase/radar-chart/animated-radar-chart.js]: that shouldn't be the desired behavior? the desired effect should that the labels style should read: {...DEFAULT_STYLE, ....USER_PROVIDED_STYLE};

regardless, this PR looks fine ill merge it when the tests pass

Comment (zachguo) [inline showcase/radar-chart/animated-radar-chart.js]: Ideally yes, but when user passes a custom `style` prop, [`style.labels` here](https://github.com/zachguo/react-vis/blob/03cfbe36b7e7c9fa018ad9b2752c712ba6e50bcd/src/radar-chart/index.js#L190) would be undefined.

**Pull request merged**